### ISSUE TITLE
Find implementations of protocol methods (cmd+alt+b)

### ISSUE THREAD

**Issue by savagematt** [cursive-ide/cursive#437]

**Comment by hlship** [comment]
This is something I would fine exceptionally valuable. Lots of great navigation end right there and suddenly you are back in text-search land.

**Comment by metametadata** [comment]
+1, I use protocols extensively and would like to quickly find all the implementations (i.e. `defrecord`s, `deftype`s, maybe even `reify`-es). Would be perfect to have the same UX as when you code in Java and see the "bubble" in the sidebar next to interface/method declaration: 

<img width="472" alt="screenshot 2017-11-17 17 11 06" src="https://user-images.githubusercontent.com/4689913/32953911-505a7f20-cbba-11e7-9423-c35d8237436f.png">

**Comment by potetm** [comment]
I want to second @metametadata's suggestion. No need to resolve individual fns. Properly finding implementations of the interface (no matter what fns happen to be implemented) would be good enough for me.

**Comment by katox** [comment]
I understand that enhancing the PSI representation (and a proper resolver) is a difficult task but supporting `defrecord`, `deftype`, `defprotocol` and `defmulti`/`defmethod` properly would ease things a lot - and it should enable more features from the platform functionality itself.

I can workaround it by expanding the expected dependency (jar) and using a **Find in Path** there but I feel like a caveman doing that.

**Comment by vincent-dm** [comment]
This feature would be very useful. I miss it every day.

**Comment by Reefersleep** [comment]
+1, I really miss this functionality, it'd be amazing to have!

**Comment by amazkovoi** [comment]
+1 This would really help!

**Comment by takaczapka** [comment]
+1 Same here

**Comment by PawelStroinski** [comment]
Yeah, this would be useful. Instead I'm just searching for `(method-name [` string.

**Comment by dlebrero** [comment]
+1

**Comment by ctomo** [comment]
+1 This would be very useful. Is there a roadmap to when we can expect this to be implemented?

**Comment by danieroux** [comment]
+1

**Comment by markbastian** [comment]
+1

**Comment by viktorkrskv0** [comment]
+1

**Comment by velios** [comment]
+1

**Comment by arnaudbos** [comment]
And another +1
